<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1251</width>
    <height>833</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SigDigger</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Icons.qrc">
    <normaloff>:/icons/icon-color.png</normaloff>:/icons/icon-color.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_11">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="mainTab">
      <property name="tabPosition">
       <enum>QTabWidget::South</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1251</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRun"/>
   <addaction name="separator"/>
   <addaction name="actionChooseProfile"/>
   <addaction name="actionSetup"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionRun">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/icons/start-capture.png</normaloff>:/icons/start-capture.png</iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </action>
  <action name="actionChooseProfile">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/icons/select-source.png</normaloff>:/icons/select-source.png</iconset>
   </property>
   <property name="text">
    <string>Choose profile</string>
   </property>
  </action>
  <action name="actionSetup">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/icons/preferences.png</normaloff>:/icons/preferences.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Icons.qrc"/>
 </resources>
 <connections/>
</ui>
